source/class/qx/tool/compiler/Es6ify.js

```diff
@@ -168,7 +168,14 @@ qx.Class.define("qx.tool.compiler.Es6ify", {
         passPerPreset: true
       };
 
-      let result = babelCore.transform(src, config);
+      let result;
+      while (true) {
+        result = babelCore.transform(src, config);
+        if (result.code === src) {
+          break;
+        }
+        src = result.code;
+      }
```

Comment:
This looks good... but is there any chance that it could get into a state where it toggles back and forth between two results, or otherwise doesn't stabilize? Maybe add a maximum number of iterations (10? 50?) as a "just in case"? I'll approve this, but I do have a bit of concern.

Reply:
Maybe even add a printed warning if it ever does hit the maximum count, so that we can look into what's causing it?

Reply:
yep, having an escape hatch sounds like a good thing ...